### DISCUSSION TITLE
Missing WinPython32-3.12 for x86

### DISCUSSION THREAD

**Discussion by Piorun2002** [winpython/winpython#1316] [Ideas]
Do you plan to continue publishing a 32-bit version, the last one available is quite old (WinPython32-3.10.11.)

**Comment by stonebig** [comment]
Hi @Piorun2002

There is no plan to do the added effort of publishing Python-32 bit wheels.
May I ask why you have still interest in Python-32 bit in 2024 ?

**Comment by Piorun2002** [comment]
I have old PC with Windows 10 x86 for Home Assistant - new version of Hass has required Python-32   >= v3.11 (yes, I know I should have switched to x64 a long time ago)

**Comment by nmarc** [comment]
We still have a few old 32-bit dlls that the manufacturer has not yet
migrated to 64-bit. Therefore, we still have to use the 32-bit Python
version.

Piorun2002 ***@***.***> schrieb am Fr., 19. Apr. 2024, 13:41:

**Comment by stonebig** [comment]
A winpython-3.12.3.0dot ?

**Comment by stonebig** [comment]
done... will add it to the webpage in a few days

**Comment by Piorun2002** [comment]
Thank you for prompt response :)

**Comment by nmarc** [comment]
if it's not too much work, it would be really cool with Qt

Piorun2002 ***@***.***> schrieb am Fr., 19. Apr. 2024, 16:42:

**Comment by stonebig** [comment]
last non-dot 32bit was, and will remain, WinPython-32bit-3.8.0.0 , october 24th 2019
